com.github.stsdc.monitor.json

```
@@ -0,0 +1,344 @@
+                "url-template": "https://github.com/libfuse/libfuse/releases/download/fuse-$version/fuse-$version.tar.gz",
+                "versions": {
+                  "<": "4.0"
+                }
+              }
+            }
+          ]
+        },
+        {
+          "name": "ostree",
+          "config-opts": [
+            "--disable-man",
+            "--with-systemdsystemgeneratordir=/app/lib/systemd/system-generators",
+            "--without-systemdsystemunitdir",
+            "--with-curl"
+          ],
+          "cleanup": [
+            "/etc/grub.d",
+            "/etc/ostree",
+            "/share/ostree",
+            "/libexec"
+          ],
+          "sources": [
+            {
+              "type": "archive",
+              "url": "https://github.com/ostreedev/ostree/releases/download/v2023.7/libostree-2023.7.tar.xz",
+              "sha256": "19cda718705f7ac8c018c939c38b1bb8412deaaa04862da98cd9fe9243f073bf",
+              "x-checker-data": {
+                "type": "json",
+                "url": "https://api.github.com/repos/ostreedev/ostree/releases/latest",
+                "version-query": ".tag_name",
+                "url-query": ".assets | first | .browser_download_url"
+              }
+            }
+          ]
+        },
+        {
+          "name": "python3-pyparsing",
+          "cleanup": [
+            "*"
+          ],
+          "buildsystem": "simple",
+          "build-commands": [
+            "pip3 install --prefix=/app pyparsing-2.4.7-py2.py3-none-any.whl"
+          ],
+          "sources": [
+            {
+              "type": "file",
+              "url": "https://files.pythonhosted.org/packages/8a/bb/488841f56197b13700afd5658fc279a2025a39e22449b7cf29864669b15d/pyparsing-2.4.7-py2.py3-none-any.whl",
+              "sha256": "ef9d7589ef3c200abe66653d3f1ab1033c3c419ae9b9bdb1240a85b024efc88b"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "name": "libwnck",
+      "buildsystem": "meson",
+      "config-opts": [
+        "-Dstartup_notification=enabled"
+      ],
+      "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/bin"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.gnome.org/sources/libwnck/3.36/libwnck-3.36.0.tar.xz",
+          "sha256": "bc508150b3ed5d22354b0e6774ad4eee465381ebc0ace45eb0e2d3a4186c925f"
+        }
+      ]
+    },
+    {
+      "name": "libgtop",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.gnome.org/sources/libgtop/2.40/libgtop-2.40.0.tar.xz",
+          "sha256": "78f3274c0c79c434c03655c1b35edf7b95ec0421430897fb1345a98a265ed2d4"
+        }
+      ]
+    },
+    {
+      "name": "libNVCtrl",
+      "build-options": {
+        "env": {
+          "LIB": "lib/x86_64-linux-gnu"
+        }
+      },
+      "no-autogen": true,
+      "no-make-install": true,
+      "build-commands": [
```

actually I misread, this should be 
```suggestion
      "post-install": [
```

and the module should be x86_64 only.